### DISCUSSION TITLE
vesamenu.c32 not a com32r image

### DISCUSSION THREAD

**Discussion by syrineGh** [Security-Onion-Solutions/securityonion#7091] [Unsupported Versions] [answered]
Hello everyone, 
After installing Security Onion on VMware i have got this message "vesamenu.c32 not a com32r image" 
even after googling it, many users used "live" or " ubuntu" as commands after they clicked on Tab to continue the installing process, but when i tired it i got this message "could not find kernel image: live" 
i would love to know how to fix this and continue my installing process.
Thank you 
Kind regards,
Syrine
![git problem](https://user-images.githubusercontent.com/98957394/152347296-0ab8bd7e-cfb5-4b83-81f0-40914f9c6f60.PNG)

**Comment by dougburks** [comment, marked as answer]
Check the ISO image using hashes or GPG:
https://securityonion.net/download
